cmd/receive_adapter/main.go

```diff
@@ -20,8 +20,14 @@ import (
 	"knative.dev/eventing/pkg/adapter/v2"
 
 	gitlabadapter "knative.dev/eventing-gitlab/pkg/adapter"
+	"knative.dev/pkg/injection"
+	"knative.dev/pkg/signals"
 )
 
 func main() {
-	adapter.Main("gitlabsource", gitlabadapter.NewEnvConfig, gitlabadapter.NewAdapter)
+	ctx := signals.NewContext()
+	cfg := injection.ParseAndGetRESTConfigOrDie()
+	ctx, _ = injection.Default.SetupInformers(ctx, cfg)
```

Comment:
@pierDipi or is there another way, to simply get one informer injected (the one with the kubeclient)?

Reply:
https://github.com/knative/eventing/blob/1ed11b74e50924e57587848152395e2ebaecfd8f/pkg/adapter/v2/main.go#L146C42-L146C59

Reply:
:+1: Thanks